backend/app/models/openai_conversation.py

```
@@ -0,0 +1,55 @@
+from sqlmodel import SQLModel, Field
+from typing import Optional
+from datetime import datetime, UTC
+
+
+class OpenAIConversationBase(SQLModel):
+    response_id: str = Field(index=True)
+    ancestor_response_id: Optional[str] = Field(default=None, index=True)
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)